include/backports.pref

```diff
@@ -0,0 +1,4 @@
+Explanation: apt: backports
+Package: *
+Pin: release a=stretch-backports, n=, v=, c=, o=, l=
```

Comment:
Suffices to say `Pin: release a=stretch-backports`. I think.

Reply:
Yeah that's probably better, I just copied what puppet put in place on our current hosts because it works and wasn't that long.